react/components/molecules/tabs/tabs.js

```diff
@@ -8,7 +8,9 @@ import { BarAnimated, ButtonTab } from "../../atoms";
 export class Tabs extends PureComponent {
     static get propTypes() {
         return {
+            state: PropTypes.object.isRequired,
             navigation: PropTypes.object.isRequired,
+            descriptors: PropTypes.object.isRequired,

```

Comment:
please remove everything not being used except state

Also, if possible please fix this typo `badgeColor={tab.bundefinedadgeColor}`